.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,12 +1,12 @@
 <!-- You must fill out this to do list for your pull request to be accepted.  If you are adding a new anime list, please follow the checklist below. Place an [x] (get rid of any spaces) inside each square as you complete each item. This is just to help you double check for any errors that might come up. 🙂 If this pull request is to address something other than adding shows or movies, please delete the text below and write your own description on what you have changed/added to the project. -->
 
-- [ ] 🔍 I have searched the `data.js` file and confirmed I am not adding a duplicate entry. Note: Different season of the same anime are okay to add such as Sword art online (2013) and Sword art online (2014) .
-- [ ] 🌻 I have added a anime year under `animeYear`. Note: Do not add ranges such as 2017-2019.
-- [ ] 🔗 I have added the myanimelist page under `animeLink`.
-- [ ] 3️⃣ I have at least three emojis listed under `animeEmoji`.
-- [ ] 5️⃣ I have a maximum of five emojis listed under `animeEmoji`.
-- [ ] 👍 My pull request has a descriptive title (such as `Added Sword Art Online` or `Added Sword Art Online: Alicization`).
-- [ ] 🖍️ I have placed the new show(s) or movie(s) in alphabetical order based on title. If the show or movie starts with 'the', then use the second word to alphabetize.
+- [x] 🔍 I have searched the `data.js` file and confirmed I am not adding a duplicate entry. Note: Different season of the same anime are okay to add such as Sword art online (2013) and Sword art online (2014) .
```

Comment:
you don't have to edit this file
this file is template on PR message

Reply:
Ah right ok